### PR TITLE
[1/7] 9910 Update compiz to 0.8.14, make it 64-bit

### PR DESCRIPTION
**Pull request by Mno-hime**
Updated all four compiz components to 0.8.14, made it 64-bit so it builds correctly with 64-bit MATE `marco`.

Later I'll separate this to 4 PRs so build server can process this correctly.

### CODE REVIEW

**Comment by Mno-hime** [comment]
It should be merged and published in this order:
1. This PR: https://github.com/OpenIndiana/oi-userland/pull/4538
2. https://github.com/OpenIndiana/oi-userland/pull/4540
3. https://github.com/OpenIndiana/oi-userland/pull/4541
4. https://github.com/OpenIndiana/oi-userland/pull/4542
5. https://github.com/OpenIndiana/oi-userland/pull/4547
6. https://github.com/OpenIndiana/oi-userland/pull/4548
7. https://github.com/OpenIndiana/oi-userland/pull/4549

**Comment by Mno-hime** [comment]
Anyway, how can one test this? I have it installed, in "Windows Preferences" I see compositing WM enabled, so this is it, or it's something completely different? (Haven't worked with compiz some 10 years...)

**Comment by pyhalov** [comment]
the easiest thing to do is to run compiz --replace

**Comment by Mno-hime** [comment]
Changed `gconftool-2` for `gsettings` so the accessibility check actually works.
```
{global} newman@lenovo:~ $ export COMPIZ_VERBOSE=yes
{global} newman@lenovo:~ $ sudo -E compiz --replace
Checking for Acessibility: 
       Disabled. 

Checking for Trusted session: 
not present.

Checking for Sunray session: 
not present. 

Checking for Composite extension: 
present. 

Checking for VNC
Running without vnc
env. variables used : 
LIBGL_ALWAYS_INDIRECT=TRUE

INTEL_BATCH=1



/usr/bin/compiz-bin (core) - Fatal: No valid GL extensions string found.
/usr/bin/compiz-bin (core) - Error: Failed to manage screen: 0
/usr/bin/compiz-bin (core) - Fatal: No manageable screens found on display :0
Cannot run compiz starting marco instead
```

**Comment by pyhalov** [comment]
0) I hope, you have NVidia adapter.
1) Test compiz-bin, not compiz
2) If compiz-bin works, fix compiz script. Likely, we should remove '--indirect-rendering' options and 
```
LIBGL_ALWAYS_INDIRECT=TRUE
  export LIBGL_ALWAYS_INDIRECT
  verbose "LIBGL_ALWAYS_INDIRECT=TRUE\n"
```
block

**Comment by Mno-hime** [comment]
ad 0) Nope, I don't have any nvidia adapter around.
ad 1) Yup.
ad 2) Indirect is gone.

Tested it on Intel Sandy Bridge from Lenovo X220. The default configuration works fine. Anyway, I guess the old compiz does not work with MATE 1.20 at all.

File: components/desktop/compiz/compiz.p5m [hunk 1]
```diff
@@ -33,15 +34,14 @@ license $(COMPONENT_LICENSE_FILE) license='$(COMPONENT_LICENSE)'
 file files/compiz.png path=usr/share/pixmaps/compiz.png
 file files/compiz-configure path=usr/lib/compiz/compiz-configure
 file files/modify-xorg-conf path=usr/lib/compiz/modify-xorg-conf
-file files/compiz.desktop path=usr/share/applications/compiz.desktop
 file files/compiz-config.glade path=usr/share/compiz/compiz-config.glade
 file files/compiz path=usr/bin/compiz mode=0555
 file usr/bin/compiz path=usr/bin/compiz-bin mode=0555 pkg.depend.bypass-generate=libGL\.so\.1
+file usr/bin/compiz-decorator path=usr/bin/compiz-decorator mode=0555
```
Comment (pyhalov) [inline components/desktop/compiz/compiz.p5m]: Move this file definition to general section

**Comment by alarcher** [comment]
Requires uninstalling desktop/compiz/plugin/compiz-fusion-main@0.8.10 on the build server.

**Comment by alarcher** [comment]
@pyhalov @Mno-hime Tested on Nvidia.

**Comment by Mno-hime** [comment]
Thanks for the testing on nvidia @alarcher.

Should I add obsolete for `compiz-fusion-main@0.8.10`? (How?)

**Comment by pyhalov** [comment]
No

**Pull request merged**